<!doctype html>
<html>
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

        <title>AFG2019 &ndash; Group 5</title>

        <link rel="stylesheet" href="css/reveal.css">
        <link rel="stylesheet" href="css/theme/maehler_dark.css">

        <!-- Theme used for syntax highlighting of code -->
        <link rel="stylesheet" href="lib/css/zenburn.css">

        <!-- Font awesome -->
        <link rel="stylesheet" href="lib/css/fontawesome.css">
        <link rel="stylesheet" href="lib/css/solid.css">
        <link rel="stylesheet" href="lib/css/brands.css">

        <!-- Custom modifications that shouldn't go into the theme -->
        <link rel="stylesheet" href="css/customise.css">

        <!-- Printing and PDF exports -->
        <script>
            var link = document.createElement( 'link' );
            link.rel = 'stylesheet';
            link.type = 'text/css';
            link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
            document.getElementsByTagName( 'head' )[0].appendChild( link );
        </script>
    </head>
    <body>
        <div class="slidelink"><a href="http://bit.ly/afg2019-5">http://bit.ly/afg2019-5</a></div>
        <div class="twitterlink"><a href="https://twitter.com/nmaehler"><i class="fab fa-twitter"></i>@nmaehler</a></div>
        <div class="reveal">
            <div class="slides">
                <section>
                    <h1>Machine learning of leaf shape GWAS</h1>
                    <p><small>A functional genomics project</small></p>
                </section>

                <section>
                    <h1>Leaf shape in aspen</h1>
                    <div style="display: flex; align-items: center;">
                        <ul>
                            <li>Large natural variation</li>
                            <li>Highly heritable</li>
                            <li>Complex trait</li>
                        </ul>
                        <img style="width: 50%; height: 50%; margin: 0 auto;" src="img/aspen_leaves.jpg" alt="Variation among aspen leaves">
                    </div>
                </section>

                <section>
                    <h1>Traditional GWAS</h1>
                    <p>We have tried doing traditional GWAS, but we have too few samples given the complexity of the trait. Also, we have only done univariate GWAS, that is, only considered one SNP at a time.</p>
                </section>

                <section>
                    <h1>Why machine learning?</h1>
                    <p>Can we find combinations of SNPs that explain the trait variation better than single SNPs?</p>
                    <p>Also, could it be possible to integrate other types of data, for example gene expression, to better describe the phenotypic variation?</p>
                </section>

                <section>
                    <h2>Data</h2>
                    <ul>
                        <li>Gene expression: ~35,000 genes, 80 genotypes (<code>htseq-count</code>)</li>
                        <li>Genetic variation: ~4 million SNPs (VCF format)</li>
                        <li>
                            Phenotype data: leaf shape (BLUPs)
                            <ul>
                                <li>Area</li>
                                <li>Indent depth</li>
                                <li>Circularity</li>
                            </ul>
                        </li>
                        <li>Gene annotations (gff3 format)</li>
                    </ul>
                </section>

                <section>
                    <h2>Software</h2>
                    <ul>
                        <li>bcftools (working with VCF files; command line)</li>
                        <li>bedtools (genome arithmetic; command line)</li>
                        <li>plink (file formats SNPs; command line)</li>
                        <li>ranger (random forests; command line/R-package)</li>
                        <li>Downstream analysis in R or Python (or any other language you like)</li>
                    </ul>
                </section>
            </div>
        </div>

        <script src="lib/js/head.min.js"></script>
        <script src="js/reveal.js"></script>

        <script>
            // More info about config & dependencies:
            // - https://github.com/hakimel/reveal.js#configuration
            // - https://github.com/hakimel/reveal.js#dependencies
            Reveal.initialize({
                dependencies: [
                    { src: 'plugin/markdown/marked.js' },
                    { src: 'plugin/markdown/markdown.js' },
                    { src: 'plugin/notes/notes.js', async: true },
                    { src: 'plugin/highlight/highlight.js', async: true, callback: function() { hljs.initHighlightingOnLoad(); } }
                ],
                history: true
            });
        </script>
    </body>
</html>
